### DISCUSSION TITLE
PHPStorm Plugin

### DISCUSSION THREAD

**Discussion by edilsoncichon** [livewire/livewire#2431] [3. Feature ideas]
It would be great if there was a plugin to assist with Livewire development.

**Comment by ju5t** [comment]
[Laravel Idea](https://plugins.jetbrains.com/plugin/13441-laravel-idea), not sure how well this works though. I'm using VS code.

source: https://twitter.com/phpstorm/status/1282630281716027394

**Reply by edilsoncichon** [reply]
That's looks great! Thank you @ju5t!
